src/satosa/backends/saml2.py

```diff
@@ -78,7 +78,9 @@ class SAMLBackend(BackendModule, SAMLBaseModule):
     """
     A saml2 backend module (acting as a SP).
     """
+    KEY_ALLOW_DISCO_INIT_CONFIG = 'allow_discovery_initiated'
     KEY_DISCO_SRV = 'disco_srv'
+    KEY_SAML_DISCOVERY_INITIATED = 'saml_discovery_initiated'
```

Comment:
@skoranda could we adopt `KEY_ALLOW_DISCO_INIT_CONFIG` only, without `KEY_SAML_DISCOVERY_INITIATED`?